test stream failed

```
--- FAIL: TestStream-4 (1.02s)
	stream_test.go:297: #0: failed to receive message from the channel
	stream_test.go:300: #0: message = {Type:MsgHup To:0 From:0 Term:0 LogTerm:0 Index:0 Entries:[] Commit:0 Snapshot:{Data:[] Metadata:{ConfState:{Nodes:[] XXX_unrecognized:[]} Index:0 Term:0 XXX_unrecognized:[]} XXX_unrecognized:[]} Reject:false RejectHint:0 XXX_unrecognized:[]}, want {Type:MsgProp To:2 From:0 Term:0 LogTerm:0 Index:0 Entries:[] Commit:0 Snapshot:{Data:[] Metadata:{ConfState:{Nodes:[] XXX_unrecognized:[]} Index:0 Term:0 XXX_unrecognized:[]} XXX_unrecognized:[]} Reject:false RejectHint:0 XXX_unrecognized:[]}
```

/cc @yichengq

--- a/rafthttp/stream_test.go
+++ b/rafthttp/stream_test.go
@@ -221,8 +221,8 @@ func TestStreamReaderDialDetectUnsupport(t *testing.T) {
 // TestStream tests that streamReader and streamWriter can build stream to
 // send messages between each other.
 func TestStream(t *testing.T) {
-	recvc := make(chan raftpb.Message)
-	propc := make(chan raftpb.Message)
+	recvc := make(chan raftpb.Message, streamBufSize)
+	propc := make(chan raftpb.Message, streamBufSize)
 	msgapp := raftpb.Message{
 		Type:    raftpb.MsgApp,
 		From:    2,
@@ -294,10 +294,10 @@ func TestStream(t *testing.T) {
 		select {
 		case m = <-tt.wc:
 		case <-time.After(time.Second):
-			t.Errorf("#%d: failed to receive message from the channel", i)
+			t.Fatalf("#%d: failed to receive message from the channel", i)
 		}
 		if !reflect.DeepEqual(m, tt.m) {
-			t.Errorf("#%d: message = %+v, want %+v", i, m, tt.m)
+			t.Fatalf("#%d: message = %+v, want %+v", i, m, tt.m)
 		}
 	}
 }

